ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+#### Subject of the issue
+Describe your issue here
+
+#### Your environment
+* version/branch
```

Comment:
Since we don't use releases this is probably not necessary. If so I would rather specify whether it's development or production